OJP/OJP_JourneySupport.xsd

```diff
@@ -502,21 +514,21 @@
 						<xs:documentation>Static place.</xs:documentation>
 					</xs:annotation>
 				</xs:element>
-				<xs:element name="TripPlace" type="TripPlaceStructure">
+				<xs:element name="TripLocation" type="TripLocationStructure">
 					<xs:annotation>
-						<xs:documentation>Place within a (moving) vehicle.</xs:documentation>
+						<xs:documentation>Location within a (moving) vehicle.</xs:documentation>
```

Comment:
Isn't that in direct contradiction to the description of *TripLocationStructure*, line 256?